eclass/distutils-r1.eclass

```diff
@@ -164,6 +166,10 @@ _distutils_set_globals() {
 				bdep+='
 					dev-python/flit_core[${PYTHON_USEDEP}]'
 				;;
+			maturin)
+				bdep+='
+					dev-python/maturin[${PYTHON_USEDEP}]'
```

Comment:
```suggestion
					dev-util/maturin[${PYTHON_USEDEP}]'
```
Formerly it didn't even install any python stuff, so guess dev-util made sense at the time. I made it install the pep517 backend after picking it up.

Reply:
Thanks for catching my thinko.